# Action.

uint32 action

uint32 GET_VERTEX_DESCRIPTOR=1
uint32 GET_EDGES_DESCRIPTORS=2
uint32 LOCALIZE_IN_VERTEX=3
uint32 LOCALIZE_EDGE=4
uint32 GET_DISSIMILARITY=5
uint32 INTERRUPT=6
uint32 CONTINUE=7

lama_interfaces/DescriptorIdentifier descriptor
---
# Result.

uint32 state
uint32 DONE=1
uint32 NOT_SUPPORTED=2

lama_interfaces/DescriptorIdentifier[] descriptors
# Data output, dependent on action.
int32[] idata
float32[] fdata
 
duration completion_time
---
# Feedback.

# Completion is -1 if unknown, 0 if just started, 1 if finised
float32 completion
duration time_elapsed
